tests/library/test_gen.py

```diff
@@ -85,11 +97,14 @@ def test_unicode(selected_model):
     # fmt: on
 
 
-def test_unicode2(selected_model):
+def test_unicode2(selected_model: models.Model):
     lm = selected_model
     prompt = "Janet’s ducks lay 16 eggs per day"
     lm += prompt + gen(max_tokens=10)
-    assert True
+    print(f"Output: {str(lm)}")
+    assert lm.metrics.prompt_tokens > 0
+    assert lm.metrics.generated_tokens > 0
+    assert lm.metrics.generated_tokens <= 10 + 1
```

Comment:
Not sure as to the reason for the off-by-one error here. Could it be token healing? Or is it just something else I'm doing wrong?